stackstorm/handler.py

```diff
@@ -144,7 +144,10 @@ def base(event, context, passthrough=False):
             except Exception:
                 LOG.warn('`event` has `body` which is not JSON')
         elif content_type == 'application/x-www-form-urlencoded':
-            event['body'] = dict(parse_qsl(['body'], keep_blank_values=True))
+            try:
+                event['body'] = dict(parse_qsl(['body'], keep_blank_values=True))
+            except Exception:
+                LOG.warn('`event` has `body` which is not x-www-form-urlencoded')
```

Comment:
Can you please also include original exception message in the warning message (`str(e.message)`).

This way it could make it easier to debug, since there are many different potential errors here which could cause an exception.

Reply:
Besides that, LGTM, will merge it.

Reply:
Absolutely! Done.